include/SDL3/SDL_stdinc.h

```diff
@@ -523,6 +523,32 @@ SDL_COMPILE_TIME_ASSERT(enum, sizeof(SDL_DUMMY_ENUM) == sizeof(int));
 extern "C" {
 #endif
 
+/**
+ * A macro to initialize an SDL interface.
+ *
+ * This macro will initialize an SDL interface structure and should be called before you fill out the fields with your implementation.
+ *
+ * You can use it like this:
+ *
+ * ```c
+ *      SDL_IOStreamInterface iface;
+ *
+ *      SDL_INIT_INTERFACE(&iface);
+ *      ...
```

Comment:
(I don't know if we currently build these on the wiki CI, but just in case, replacing `...` with `// initialize your interface here` might be good.)